app/models/course/assessment/submission.rb

```diff
@@ -22,6 +22,7 @@ class Course::Assessment::Submission < ApplicationRecord
       event :publish, transitions_to: :published
     end
     state :graded do
+      event :unsubmit, transitions_to: :attempting
```

Comment:
> I think I am good to merge this PR, but I just have some minor nits:
> 
>     * Is it possible to rename the `custom` and `confirmCustom` to something more semantic? Maybe something like `primaryConfirm` and `secondaryConfirm` would be good. Or `primaryAction` and `secondaryAction` (but the component is called `ConfirmationDialog`, so not sure if this is ideal). I'm open to suggestions for this.
> 
>     * (UX nit) It would be good if the confirmation flash for the instructor includes info on the unsubmission result. For example, the below flash message would be better as "Question 1 has been successfully switched to MRQ, and all submissions to this assessment have been unsubmitted".
>        <img alt="Screenshot 2021-10-05 at 6 01 15 PM" width="1101" src="https://user-images.githubusercontent.com/45617494/136002545-3f8753bf-c651-4660-8f5a-615f1d28db07.png">
> 
>     * (UX nit) Maybe not in this PR, but it would be good to think of ways we can make it more explicit to the student that their submission was unsubmitted AND which question was changed. This will also help make it clearer to the students as to why their answer last time is suddenly gone.

- I prefer to keep 'confirm' as it is the default naming for the modal and it's used everywhere. Perhaps we can use additionalConfirm instead of customConfirm?
- Sure can do
- Sure, i was thinking we can send them an email notification to inform them for this?

Also, i will removed the unsubmit workflow from the graded state. Instead, if a submission is graded, i will ungrade it and unsubmit it instead.